code/mob/living.dm

```diff
@@ -499,7 +499,7 @@
 				grace_penalty = time_left
 
 		if (target == equipped)
-			equipped.attack_self(src, params, location, control)
+			equipped.AttackSelf(src, params, location, control)
```

Comment:
Hm. Should we add these arguments to AttackSelf? Or should we delete them from this call?

Reply:
I thought about adding the arguments, but nothing uses `params`, `location`, or `control`.  Anything using those create a different UX for mouse interaction vs keyboard interaction.  There could be some interesting use cases of looking at what pixels were interacted with but at this point that would be such a departure that the addition should warrant more scrutiny.